package.json

```diff
@@ -21,10 +21,14 @@
     "type": "git",
     "url": "https://github.com/turbo87/sidebar-v2.git"
   },
+  "main": "./examples/ol5.js",
   "scripts": {
-    "test": "gulp lint"
+    "test": "gulp lint",
+    "start": "parcel ./examples/ol5.html",
+    "build": "parcel build --public-url ./ ./js/ol5-sidebar.js"
```

Comment:
we already use gulp for the basic build stuff. why do we need parcel?

Reply:
Well, I used it because I knew how to use it and not gulp...

Hopefully, there seems to be [this guide](https://pxpx.co.uk/blog/article/compiling-es6-with-gulp-babel) that says that it is possible "to compile ES6 and beyond into browser-compliant ES5". It seems to be using [gulp-babel](https://www.npmjs.com/package/gulp-babel).

If you want I can dig into this and see what it can do to this regard.